### DISCUSSION TITLE
Regenerate Filer Tree/Store

### DISCUSSION THREAD

**Discussion by ProjectInitiative** [seaweedfs/seaweedfs#2333] [Q&A] [answered]
Reading through [the wiki](https://github.com/chrislusf/seaweedfs/wiki/Migrate-to-Filer-Store) I have a couple of questions:

* Am I able to regenerate a filer store from scratch? AKA re-index if the original is lost? 
* If so, is that what the `weed filer.copy` command aims to achieve?

**Comment by chrislusf** [comment, marked as answer]
> * Am I able to regenerate a filer store from scratch? AKA re-index if the original is lost?

Not.

You can backup filer store with common backup tools, or in this way https://github.com/chrislusf/seaweedfs/wiki/Async-Filer-Metadata-Backup to a different store. If the original filer store failed, you can switch to the backup one fairly easily.

**Reply by ProjectInitiative** [reply]
Gotcha, so losing your filer stores means losing access to data uploaded through a filer, so backups are key here.
